src/components/custom-field-input-number/__snapshots__/custom-field-input-number.test.jsx.snap

```diff
@@ -40,11 +34,11 @@ exports[`CustomFieldInputNumber has defaults 1`] = `
 
 exports[`CustomFieldInputNumber has defaults 2`] = `
 Object {
-  "dirty": false,
+  "dirty": true,
```

Comment:
Why is the field dirty by default?

Reply:
I think it is. That's something I'll have to dig at